service_test.go

```diff
@@ -244,6 +244,25 @@ func TestCreateService(t *testing.T) {
 	autopilot.Equals(t, 1, len(result.Aliases))
 }
 
+func TestCreateServiceWithParentSystem(t *testing.T) {
```

Comment:
I duplicated both the create and update tests which because they still pass show that we can generate an update input without the `parent` field and then this shows we can generate an input with the `parent` field.

We can come back around later to shore up the copy paste once we take on "table tests"